packages/next/src/server/lib/trace/tracer.ts

```diff
@@ -17,10 +17,14 @@ let api: typeof import('next/dist/compiled/@opentelemetry/api')
 // @opentelemetry/tracing that is used, and we don't want to force users to use
 // the version that is bundled with Next.js.
 // the API is ~stable, so this should be fine
-try {
+if (process.env.NEXT_RUNTIME === 'edge') {
   api = require('@opentelemetry/api')
-} catch (err) {
-  api = require('next/dist/compiled/@opentelemetry/api')
+} else {
+  try {
+    api = require('@opentelemetry/api')
+  } catch (err) {
+    api = require('next/dist/compiled/@opentelemetry/api')
+  }
 }
```

Comment:
This looks like a behavior change that it no longer fallbacks to the builtin opentelemetry lib under Edge?

Reply:
It’s already aliased the pre compiled one to that one if it’s not installed. This was try to avoid both are bundled for edge

Reply:
I saw that alias applied to non-Edge runtimes as well so why not just do `api = require('@opentelemetry/api')` for both and remove the try-catch and runtime condition?

Reply:
It was also for RSC server layer, but for nodejs where it’s not bundled we need to keep the try-catch, e.g. for next server